core/utils/keycodes.js

```diff
@@ -166,3 +167,5 @@ Blockly.utils.KeyCodes = {
   // http://en.community.dell.com/support-forums/laptop/f/3518/p/19285957/19523128.aspx
   PHANTOM: 255
 };
+
+exports = KeyCodes;
```

Comment:
Is this exporting `Blockly.utils.KeyCodes.KeyCodes` now instead of `Blockly.utils.KeyCodes`?

Christopher's write-up makes me think so, but I'm really not sure:
> ```
> // Following goog.provides('MyModule'):
> /** @const */
> MyModule.BLUE = '#0000ff';
> /** @param ...  @return ... */
> MyModule.colourNameToHex(name) { ... };
> ```
> becomes:
> ```
> /** @const */
> const BLUE = '#0000ff';
> /** @param ...  @return ... */
> function colourNameToHex(name) { ... };
> 
> exports = {BLUE, colourNameToHex};
> ```

Reply:
My example may be a bit misleading, because I was treating `BLUE` as just an arbitrary constant rather than a member of an enum.  I will try to improve it.

@moniika's code looks correct.  As written, the return value of `goog.require('Blockly.utils.KeyCodes')` will be the `KeyCodes` enum object, which will be compatible with what happened before this commit.  That means code that uses this module can either do `const KeyCodes = require('...KeyCodes');  let code = KeyCodes.BACKSPACE` or `const {BACKSPACE} = require('...KeyCodes'); let code = BACKSPACE;`.